impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.conf.js

```diff
@@ -669,7 +679,12 @@ define(function() {
             yAxisSizeMax: 80, // shouldn't it be: maxHorizontalTextWidth ??
 
             // . dot
-            dot_lineWidth: function() { return this.finished(0); },
+            dot_lineWidth: function() {
+              // show nullShape
+              var v = this.getSize() != null ? 0 : this.delegate();
```

Comment:
I know the above one is needed, as the bubble chart acts as scatter when the size isn't bound, and shows a border around the bubbles if the bound is not checked. It probably makes sense to be here also.